ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -12,43 +12,61 @@
   <h3>{{t 'resources.worker.form.steps.1.title'}}</h3>
   <div class='worker-create-section'>
     {{#if (feature-flag 'byow-pki-hcp-cluster-id')}}
-      <form.input
-        @name='cluster_id'
+      <Hds::Form::TextInput::Field
```

Comment:
Aren't we missing attributes here such as `isInvalid` or `isOptional`? if you think we do not, can you please elaborate?

Reply:
@calcaide the only field that returns an error on creating a `-worker-generated-auth-token`, all other fields are output only.  However they are optional, in which I fixed in latest commit.  The error messaging for `-worker-generated-auth-token` is also fixed in next commit.